scripts/get_token.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This script will connect to the vCD using username and password,
+# and show the header that cointains an authorization token.
+#
+user=$1
+password=$2
+org=$3
+IP=$4
+
+if [ -z "$IP" ]
+then
+    echo "Syntax $0 user password organization IP_address"
```

Comment:
Can one use hostname here? If yes, I suggest we say "hostname" here as there should be more cases where user has hostname as opposed to IP (think of all the tenants renting the vCDs at SPs).

Reply:
updated